subsys/mpsl/cx/Kconfig

```diff
@@ -4,8 +4,20 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
+DT_COMPAT_CX_NRF700X := nordic,nrf700x-coex
+# config MPSL_CX_SUPPORTED
+
+config MPSL_CX_ANY_SUPPORT
+	bool
+	default $(dt_nodelabel_enabled,nrf_radio_coex)
```

Comment:
Is this Kconfig necessary?

Can't we use  $(dt_nodelabel_enabled,nrf_radio_coex) directly?

Reply:
Same for the below option.

Reply:
I think it is more readable with the Kconfig. Also, this way it mirrors the way it is done in `subsys/mpsl/fem`